Doc/library/unittest.mock.rst

```diff
@@ -303,14 +303,14 @@ the *new_callable* argument to :func:`patch`.
 
     .. method:: assert_called_once_with(*args, **kwargs)
 
-       Assert that the mock was called exactly once and with the specified
-       arguments.
+       Assert that the mock was called exactly once and that that call was
```

Comment:
"that that"? "that the call"? "that call"?

Sorry, english is not my first language, so I don't know if it's correct or not!